doc/contributing/advocacy-ambasador-program.md

```diff
@@ -130,3 +130,39 @@ or the information to be shared.
 Add a list of GitHub handles for those within the project that
 have volunteered to be contacated when necessary by ambassadors
 to get more info about the message to be promoted.
+
+#### Node.js is a great choice for a JavaScript runtime
+
+##### Goal
+
+Highlight the benefits of chosing Node.js as your backend JavaScript runtime. Focus on what is great
+about Node.js avoiding comparisons with other options. We don't want to say negative things about
+other options, only highlight what is great about Node.js as a choice
+
+Some of the things to highlight include:
+
+* How widely it is used (you never get fired for chosing Node.js).
+* The openess of the project. It is part of the OpenJS Foundation and its's governance is set up to avoid
+  any one company from dominating the project. Decisions are made by the collaborators (of which there are quite
+  a few) versus a small
+  number of people.
+* It has predictable and stable releases and has delivered on the release schedule for many years.
```

Comment:
Is it better to call out specific year?
```suggestion
* It has predictable and stable releases and has delivered on the release schedule since 2015.
```